app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/LearningSettingsActivityTest.kt

```diff
@@ -63,7 +69,12 @@ class LearningSettingsActivityTest {
     }
 
     @Test
-    fun pressingRepresentationButtonDoesNothing() {
-         assertNotClickable(R.id.learningSettings_btChooseRepresentation)
+    fun presentationModeWorks(){
+
+    }
+
+    @Test
+    fun representationModeWorks(){
+
     }
```

Comment:
What is the goal of those empty Tests? Forgot to delete them? If not, maybe comment to explain why they are empty.

Reply:
yup, forgot